src/strategies/moonbase/README.md

```diff
@@ -0,0 +1,36 @@
+# Moonbase
+
+This is the strategy, it returns the balances of the voters for MBG token balances
+in Moonbase project(pools, farms, vaults, token).
+
+Here is an example of parameters:
+
+```json
+[
+  {
+    "name": "Example query Moonbase",
+    "strategy": {
+      "name": "moonbase",
+      "params": {
+        "address": "0xc97c478Fc35d75b51549C39974053a679A5C67E1",
+        "masterChef": "0x830304d6C669d33738c7E4c1F2310CC1E530Df63",
+        "moonbaseLPs": [
+          {
+            "address": "0x230C64C42886A1F6b91eD8C11B59a2D45865d38F",
+            "pid": 7
+          }
+        ],
```

Comment:
Could you add a limit to `moonbaseLPs` to avoid memory issues, return an error of add a JSON schema

Reply:
How can I fix this error?

Reply:
actually all tests passed. So what should I do for this issue?